packages/libs/error-stack/src/ser.rs

```diff
@@ -0,0 +1,152 @@
+//! Implementation of general report serialization.
+//!
+//! The value can be of any type, currently only printable attachments and context are supported, in
+//! the near future any values will be supported through the use of hooks.
+//!
+//! The returned JSON returned is a list of all current sources with the following output:
```

Comment:
Serialization is not only JSON
```suggestion
//! The serialized [`Report`] is a list of all current sources with the following output:
```